src/sentry/static/sentry/app/components/organizations/timeRangeSelector/index.jsx

```diff
@@ -125,7 +123,15 @@ class TimeRangeSelector extends React.Component {
                 value={relative}
               />
             )}
-
+            {shouldShowBoth && (
+              <CombinedSelector
```

Comment:
Do we need a new component for combined? Can we just show both, the absolute and relative components?

Reply:
I don't know if we'll even need absolute and relative separately in future and could delete them, but don't want to change those until we decide, wanted to keep this separate for now.